stream/events.go

```diff
@@ -54,6 +40,18 @@ func (e *Events[A]) Record(event Event[A]) {
 	})
 }
 
+// PushEvents loads a set Events with version set to the last event's version.
+func (e *Events[A]) PushEvents(events []VersionedEvent[A]) error {
```

Comment:
I feel `Load` captures better what is happening here - the events are loaded from the storage into the stream. Pushing sounds more like appending to the and of the queue, and that's not what's happening here - the queue is replaced.

Reply:
no problem for me. Let's go with Load.

Reply:
https://github.com/ThreeDotsLabs/esja/pull/14/commits/37fb7359b73b8259eae4fb51d6bb96b2ae322d7f